### PR TITLE
[fix](schema scanner) fix query some schema table report invalid parameter

### PR DESCRIPTION
**Pull request by Cai-Yao**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Example:
```
SELECT ROUTINE_SCHEMA AS PROCEDURE_CAT, NULL AS PROCEDURE_SCHEM,ROUTINE_NAME AS PROCEDURE_NAME,NULL AS NUM_INPUT_PARAMS,NULL AS NUM_OUTPUT_PARAMS,NULL AS NUM_RESULT_SETS,ROUTINE_COMMENT AS REMARKS,IF(ROUTINE_TYPE = 'FUNCTION', 2,IF(ROUTINE_TYPE= 'PROCEDURE', 1, 0)) AS PROCEDURE_TYPE FROM INFORMATION_SCHEMA.ROUTINES WHERE ROUTINE_SCHEMA = DATABASE();
```
```
ERROR 1105 (HY000): errCode = 2, detailMessage = invalid parameter
```

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Cai-Yao** [comment]
run buildall

File: be/src/exec/schema_scanner.cpp [hunk 1]
```diff
@@ -84,10 +84,6 @@ Status SchemaScanner::init(SchemaScannerParam* param, ObjectPool* pool) {
         return Status::InternalError("invalid parameter");
     }
 
-    if (_columns.empty()) {
-        return Status::InternalError("invalid parameter");
```
Comment (yiguolei) [inline be/src/exec/schema_scanner.cpp:88]: add the test sql to regression test pls.

**Review by yiguolei — APPROVED** [review]
lgtm

**Comment by yiguolei** [comment]
run buildall

**Pull request merged**